### DISCUSSION TITLE
Stylelint Filename

### DISCUSSION THREAD

**Discussion by juliankasimir** [eviden-parallel/community#13] [General]
Until [Rome](https://rome.tools/) also supports HTML, CSS, we are still using [Stylelint](https://stylelint.io/), [Eslint](https://eslint.org/), [Prettier](https://prettier.io/).

In the meantime, I would like to suggest the following:

File name:
`.stylelintrc` -> `.stylelintrc.json`

Why?

[Citation](https://stylelint.io/user-guide/configure/):

> The .stylelintrc file (without extension) can be in JSON or YAML format. We recommend adding an extension to help your editor provide syntax checking and highlighting.

**Comment by Th3S4mur41** [comment]
Sounds good, this would require an update in each repository
Following up in https://github.com/eviden-parallel/stylelint-config/issues/171
